src/app/api/autofill/route.ts

```diff
@@ -0,0 +1,18 @@
+import { NextResponse } from 'next/server'
+
+export async function GET() {
+  const options = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  }
+
+  const res = await fetch(
+    'https://api-beta.copilot.com/v1/custom-fields',
```

Comment:
@aatbip looking at Next doc, seems like Next js does some kind of caching for `fetch`, are we aware of the effects?

Reply:
This is the api route. I have done caching while using this route in the server component [HERE](https://github.com/pagevamp/copilot-client-home/blob/ad749097562cc84e280a132b2bef225b30584161/src/app/page.tsx#L4)